sqlite-mode-extras.el

```diff
@@ -57,6 +57,9 @@
 (require 'sqlite-mode)
 (require 'view)
 
+(defvar sqlite-mode-extras-auto-complete-enabled t
```

Comment:
Since intended for user customization, we should use defcustom.

Reply:
PR should be updated, I don't write a lot of elisp so thanks for the hints.